lambdas/webhook_processor/webhook_processor.py

```diff
@@ -0,0 +1,163 @@
+import base64
+import json
+import os
+import urllib
+
+from botocore.vendored.requests import post
+
+import logging
+logger = logging.getLogger()
+
+# First log the function load message, then change
+# the level to be configured via environment variable.
+logger.setLevel(logging.INFO)
+logger.info('Loading function')
+
+# Log level is set as a string, default to 'INFO'
+log_level = os.environ.get('LOG_LEVEL', 'INFO').upper()
+numeric_level = getattr(logging, log_level, None)
+if not isinstance(numeric_level, int):
+    raise ValueError('Invalid log level: {}'.format(log_level))
+logger.setLevel(numeric_level)
+
+
+def _get_target_urls():
+    """
+    Get the target URLs for the processed hooks from a comma delimited list of URLs,
+    set in an OS environment variable.
+
+    Return a list of target URLs
+    """
+    url = os.environ.get('TARGET_URLS')
+    if not url:
+        raise StandardError('Environment variable TARGET_URLS was not set')
+
+    return url.split(',')
```

Comment:
Yes, I was intending for the same payload to go to different endpoints.
I am considering buildWithParameters calls out of scope.

I had thought that we needed to send to both the ghprb plugin and the github plugin endpoints, but I'm looking at the hook settings for the edx-platform repo and I only see that going to the ghprb one. Let's review them together, I think you might have better insight into this.